### PR TITLE
syno-magnet: Update package location because of renamed package name

### PR DESCRIPTION
**Pull request by neykov**
The package name has been changed so we need to update the web app url. Override the 'config' coming from the app with the proper location.

_Motivation:_ The package name changed in  https://github.com/SynoCommunity/spksrc/commit/4eacfb652c96339a992af705d43677b9bf7bc9a7 which caused the url to change. The app assumes a different URL so the PR overrides the app assumption with the correct location.

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

**Comment by neykov** [comment]
cc @ymartin59 - see changes here for further updates needed for https://github.com/SynoCommunity/spksrc/commit/4eacfb652c96339a992af705d43677b9bf7bc9a7.

**Comment by ymartin59** [comment]
Sorry for that side-effect... I have not taken care enough. I changed name for homogeneity with all other packages, package name is expected to be consistent with "spk/" folder name and only minor cases and dashes are used.

File: spk/syno-magnet/Makefile [hunk 1]
```diff
@@ -44,10 +47,24 @@ include ../../mk/spksrc.extract.mk
 
 .PHONY: syno_magnet_copy
 
-# Icons already included in download
-$(ICON_DIR): ;
+# Override because we need type: legacy
```
Comment (ymartin59) [inline spk/syno-magnet/Makefile]: Please add SERVICE_TYPE support with default value "url" in spk.service.mk directly at https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.service.mk#L214

Comment (ymartin59) [reply spk/syno-magnet/Makefile]: I changed my mind about variable name, SERVICE_URL_TYPE seems more appropriate to me.

Comment (ymartin59) [reply spk/syno-magnet/Makefile]: Correct.

**Pull request merged**